### ISSUE TITLE
cw

### ISSUE THREAD

**Issue by 1wsx10** [chaoren/vim-wordmotion#69]
ok so there are two things i have noticed.

thing 1:
` asdf_`
`^     ` cursor
cw here will change up to the underscore. default vim cw will change up to the a (it goes up to the word, not to the end of the word)
 
thing 2:
vims special case for cw (see https://old.reddit.com/r/vim/comments/26nut8/why_does_cw_work_like_ce/)
it would be nice if there was an option for the more canonical version of cw

**Comment by chaoren** [comment]
I fixed this a while back. Try updating the plugin?

**Comment by chaoren** [comment]
Oh, the cursor is on the space before the `a` in the first case. Yes, that looks like a bug.

I can't reproduce the second case.

**Comment by chaoren** [comment]
Yes, I know. I specifically reproduced the behavior of `cw` because people asked for it, so it should be there. You probably have an oudated version of the plugin if that behavior is not happening for you.

**Comment by tuurep** [comment]
It would be elegant if the cpoptions preference for this would be respected

In Vim, this is

```
set cpoptions-=z
```

(note: lowercase)

But in Neovim, it's

```
set cpoptions-=_
```

Docs:

Vim: https://vimhelp.org/options.txt.html#cpo-z
Neovim: <https://neovim.io/doc/user/options.html#cpo-_>

Oh yes also, I reproduced your "Thing 1". I'm interested in this issue as a whole.
